.github/workflows/omuse-ci.yml

```diff
@@ -41,122 +45,40 @@ jobs:
         ./generate_packages.sh
 
         cd omuse-framework
-        echo "::set-output name=VERSION::$(python setup.py --version)"
+        omuse_version=$(python setup.py --version | head -n1 | awk '{print $1;}')
+        echo "VERSION=$omuse_version" >> $GITHUB_OUTPUT
 
-    - uses: actions/upload-artifact@v2
+    - uses: actions/upload-artifact@v3
       with:
         name: sdists-${{ steps.sdists.outputs.VERSION }}
         path: packages/dist/*.tar.gz
 
-  omuse-era5:
```

Comment:
Maybe it would be good if you walk me through these changes and why you made them in person. I haven't looked at the omuse CI before and it's quite hard to understand where we are coming from and what it is that you solved.